KomaMRICore/src/simulation/SimulatorCore.jl

```diff
@@ -409,6 +409,8 @@ function simulate(
         out = Xt
     elseif sim_params["return_type"] == "mat"
         out = sig
+    elseif sim_params["return_type"] == "full"
+        out = Dict("state" => Xt, "mat" => sig)
```

Comment:
I like this change (I would prefer to output a Tuple), but it is not relevant to this PR.